pom.xml

```diff
@@ -426,7 +426,7 @@
 
 					<!-- Third argument prevents Java from popping up lots of windows on
 						MacOS -->
```

Comment:
after thinking about this a bit more - could it be that the 8GB are only required to cover an existing memory leak (maybe in mlcontext or spark internals)? Even without these new tests, our testsuite already has an issue of memory leaks when run in a single JVM process but I was not able to really resolve them so far. These issues don't show up in our nightly builds because, as @deroneriksson mentioned, maven uses an individual process per suite.